content-src/lib/getVideoPreview.js

```diff
@@ -1,32 +0,0 @@
-const getYouTubeID = require("get-youtube-id");
```

Comment:
Sure. I tried to see if there were any other unused dependencies in `package.json` and looks like this was the only one.